tests/test_project_user_keys.py

```diff
@@ -253,3 +410,61 @@ def test_share_project_keys_via_two_invites(client):
             number_of_asserted_projects += 1
     assert len(project_invite_keys) == number_of_asserted_projects
     assert len(project_invite_keys) == 5
+
+
+def test_update_user_keys_for_password_change(client):
+    user = models.User(username="randomtestuser", password="password")
+
+    public_key_initial = user.public_key
+    nonce_initial = user.nonce
+    private_key_initial = user.private_key
+    kd_salt_initial = user.kd_salt
+
+    assert public_key_initial is not None
+    assert nonce_initial is not None
+    assert private_key_initial is not None
+    assert kd_salt_initial is not None
```

Comment:
`is not None` is not needed, but also no biggie

Reply:
changed!